src/models/parallelCoordinatesChart.js

```diff
@@ -50,6 +50,20 @@ nv.models.parallelCoordinatesChart = function () {
             }
         };
 
+        tooltip.contentGenerator(function(data) {
```

Comment:
Display a new tooltip for lines:
![tooltipline](https://cloud.githubusercontent.com/assets/6142727/11661843/2bd99192-9dd4-11e5-87b6-b0f010672edd.jpg)
 And for dimensions:
![tooltipdim](https://cloud.githubusercontent.com/assets/6142727/11661916/77c406be-9dd4-11e5-81e2-4ec74af7a692.jpg)